onnxscript/backend/onnx_export.py

```diff
@@ -152,7 +152,10 @@ def _translate_value_info(value_info: ValueInfoProto) -> str:
 
 def _to_str(s):
     if isinstance(s, bytes):
-        return s.decode("utf-8")
+        try:
+            return s.decode("utf-8")
+        except UnicodeDecodeError:
+            return s.decode("latin1")  # or "cp1252" or other fallback
```

Comment:
I have seen cases where the string attributes iare used to store binary data (e.g. in the custom Tokenizer op, even though this is invalid onnx). Is this trying to address those cases? If so, I would simply preserve the b-string to avoid accidental modifications.

Related: https://github.com/onnx/ir-py/pull/184

Reply:
Right - that's the same model I looked at. Making a suggestion